docs/admin/access.rst

```diff
@@ -79,46 +69,47 @@ Project access control
 .. note::
 
     By turning on access control, all users are prohibited from accessing anything
-    within a given project, unless you add the permissions for them to do just that.
+    within the given project, unless you add the permissions for them to do just that.
 
 .. include:: /snippets/not-hosted-libre.rst
 
 Limit user's access to individual projects by selecting a different access
-control variation on the :guilabel:`Access` tab in the :guilabel:`Settings` of
```

Comment:
Why variation? How many are there?

Reply:
Partly, but "variation" suggests it is a custom set of stuff that results in a variation, and "type" sounds like it is a list of pre-determined ways of control to chose from.

Adjust vs. pick.